File: BibliotecaApi/BibliotecaDLL/Loans/Loan.cs
using MongoDB.Bson.Serialization.Attributes;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BibliotecaDLL.Loans
{
    [BsonIgnoreExtraElements]
    public class Loan
    {
        public string UserId { get; set; }
        public int BookId { get; set; }
        public DateTime Borrowed { get; set; }
        public DateTime? Returned { get; set; }

        public Loan(string userId, int bookId, DateTime borrowed, DateTime? returned) {
            this.UserId = userId;
            this.BookId = bookId;
            this.Borrowed = borrowed;
            this.Returned = returned;
        }

    }
}

File: BibliotecaApp/src/models/Loan.ts
export class Loan {
    public bookId: number;
    public userId: String;
    public returned: Date|null;
    public borrowed: Date;

    constructor(bookId: number, userId: String, returned:Date|null, borrowed:Date) {
        this.bookId = bookId;
        this.userId = userId;
        this.returned = returned;
        this.borrowed = borrowed;
    }
}